_computing/access_methods.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Computing Resource Access Methods
+last_modified_at: 2018-09-27
+main_authors:
+primary_reviewers:
+---
+>Note:  This page is still a work in progress.  If you have specific input on topics related to access methods, please email them to `sciwiki`.
+
+## Client Devices
+
+Client devices are supported by Center IT (ADM, CRD, VIDD) as well as Division IT groups (BSD, HB, PHS).  
+
+### Laptops and Desktops
+
+Center IT supports 3 different operating systems:
+
+- Laptops: 
+    - Windows
+    - MacOS
+- Desktops: 
+    - Windows
+    - MacOS
+    - Ubuntu Linux LTS
+
+Please see Center IT's page in CenterNet about [Laptops and Desktops](https://centernet.fredhutch.org/cn/u/center-it/help-desk/laptops-and-desktops.html) for more information about suppport.  
+
+>Note: If you request a Linux Desktop, Center IT will recommend that you use NoMachine from a Windows or Mac system instead as this is the most appropriate choice for most users but not all. If it is required for your work, it can be supported.  
+
+### Mobile Devices
+
+Please see Center IT pages for more informaiton on [Mobile Device Services](https://centernet.fredhutch.org/cn/u/center-it/services/mobile_device_service.html) or contact [Help Desk](https://centernet.fredhutch.org/cn/u/center-it/help-desk.html) with questions or problems.  
+
+
+## Access Methods (Terminals, X11, ssh & tmux)
+
+There are multiple ways you can get access Scientific Computing resources which are all running on a supported version of Ubuntu Linux. The most simple form of access is using a secure shell terminal software such as `ssh` or `putty`. You may also need graphical output (GUI), for example to use tools like R Studio or advanced text editors. For this you will need X11 tools which can either be installed as add-ons to your ssh program or a dedicated tool such as NoMachine.
+
+### Putty (Windows)
+
+Putty is the most widely used secure shell software for Windows. You can install putty through "Software Center" and then simply connect to host name `rhino`
+
+
+### iTerm2 (Mac OSX)
+
+The default Mac terminal is a good choice for connecting to and using SciComp Linux-based resources. If you are using to using the features of `screen` we recommend using [iTerm2](https://www.iterm2.com/) for your local terminal program on your Mac.  iTerm2 is fully integrated with tmux. Tmux provides most of the same features as GNU Screen.  If you run tmux on `rhino` you must remember on which of the 3 rhinos you ran it and the connect directly to that machine using SSH.  
```

Comment:
@dirkpetersen  what is this random code chunk telling the reader? Also, what is tmux?